packages/key-per-file-configuration/src/index.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) TotalSoft.
+// This source code is licensed under the MIT license.
+
+import * as chokidar from 'chokidar'
+import * as path from 'path'
+import * as fs from 'fs'
+import * as glob from 'glob'
+
+const { KEY_PER_FILE_CONFIG_PATH } = process.env
+const defaultConfigPath = 'runtime'
+
+export interface Options {
```

Comment:
Interfaces, enums and type aliases should be defined in a `types.ts` file. This is a organisational approach we decided to follow in this project, inspired by the 'Component-based' structure (read more [here](https://www.becomebetterprogrammer.com/typescript-organizing-and-storing-types-and-interfaces/) ), a component in our case represents a package. 
We should maintain the same structure throughout the packages developed in this monorepo.